editors/vscode/package.json

```diff
@@ -82,6 +90,11 @@
         "title": "Restart server",
         "category": "Slint",
         "icon": "$(debug-restart)"
+      },
+      {
+        "command": "slint.newProject",
+        "title": "Create New Project",
```

Comment:
I suggest to be explicit that this will get a template:

```suggestion
        "title": "Create New Project from a Template",
```

Reply:
Good point